core/shared/src/main/scala-2.13/fs2/ChunkPlatform.scala

```diff
@@ -0,0 +1,45 @@
+package fs2
+
+import scala.collection.immutable.ArraySeq
+import scala.collection.immutable
+import scala.reflect.ClassTag
+
+trait ChunkPlatform[+O] { self: Chunk[O] =>
```

Comment:
Can you make this trait and it’s companion `private[fs2]` both here and in 2.13 file? Otherwise looks ready for merge. Thanks!